src/main/java/su/litvak/chromecast/api/v2/ChromeCasts.java

```diff
@@ -85,10 +86,31 @@ public void serviceRemoved(ServiceEvent event) {
                 }
             }
         }
+        }
+
+        @Override
+        public void serviceResolved(ServiceEvent se) {
+            throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
Ugh, thanks for pointing me to this, this was NetBeans autoinsert code. Will fix.